lib/Backends/CPU/CPUBackend.h

```diff
@@ -47,6 +47,9 @@ class CPUBackend : public BackendUsingGlowIR {
 
   std::unique_ptr<CompiledFunction> compile(Function *F,
                                             const Context &ctx) const override;
+  /// Loads the function binary and constants onto device.
+  /// For CPU case this is primarily just allocating constant memory.
+  void loadFunctionOnDevice(const Context &ctx);
```

Comment:
I believe this API should be in a different class. Like DeviceManager that each backend needs to implement.
Basically, a backend should come with different and separate APIs, at least two, compile and runtime.

Reply:
So that particular function was meant to be deleted :P But! I agree I created #1951 to track designing and creating an API for runtime.